### PR TITLE
Organize VisualFSharp UnitTest Files

### PR DESCRIPTION
**Pull request by cloudRoutine**
Collects language service test source files into `LanguageService`
Collects project system test source files into `ProjectSystem`

### CODE REVIEW

**Comment by vasily-kirichenko** [comment]
@cloudRoutine welcome back :)

File: vsintegration/tests/unittests/VisualFSharp.UnitTests.fsproj [hunk 1]
```diff
@@ -38,26 +38,26 @@
...
-    <Compile Include="Tests.LanguageService.ErrorRecovery.fs" />
-    <Compile Include="Tests.ProjectSystem.Configs.fs" />
-    <Compile Include="Tests.ProjectSystem.Miscellaneous.fs" />
-    <Compile Include="Tests.ProjectSystem.MultiTargeting.fs" />
-    <Compile Include="Tests.ProjectSystem.ProjectItems.fs" />
-    <Compile Include="Tests.ProjectSystem.Project.fs" />
-    <Compile Include="Tests.ProjectSystem.References.fs" />
-    <Compile Include="Tests.ProjectSystem.RoundTrip.fs" />
-    <Compile Include="Tests.ProjectSystem.UpToDate.fs" />
+    <Compile Include="LanguageService\Tests.LanguageService.General.fs" />
```
Comment (cartermp) [inline vsintegration/tests/unittests/VisualFSharp.UnitTests.fsproj]: Could this get renamed to `LegacyLanguageService`?

File: vsintegration/tests/unittests/VisualFSharp.UnitTests.fsproj [hunk 2]
```diff
@@ -38,26 +38,26 @@
...
+    <Compile Include="LanguageService\Tests.LanguageService.GotoDefinition.fs" />
+    <Compile Include="LanguageService\Tests.LanguageService.NavigationBar.fs" />
+    <Compile Include="LanguageService\Tests.LanguageService.ParameterInfo.fs" />
+    <Compile Include="LanguageService\Tests.LanguageService.QuickInfo.fs" />
+    <Compile Include="LanguageService\Tests.LanguageService.QuickParse.fs" />
+    <Compile Include="LanguageService\Tests.LanguageService.Script.fs" />
+    <Compile Include="LanguageService\Tests.LanguageService.TimeStamp.fs" />
+    <Compile Include="LanguageService\Tests.LanguageService.ErrorList.fs" />
+    <Compile Include="LanguageService\Tests.LanguageService.ErrorRecovery.fs" />
+    <Compile Include="ProjectSystem\Tests.ProjectSystem.Configs.fs" />
```
Comment (cartermp) [inline vsintegration/tests/unittests/VisualFSharp.UnitTests.fsproj]: `LegacyProjectSystem` as well

File: vsintegration/tests/unittests/VisualFSharp.UnitTests.fsproj (hunk 1 above)
Comment (cloudRoutine) [reply vsintegration/tests/unittests/VisualFSharp.UnitTests.fsproj]: sure thing 👍

**Pull request merged**